conf.py

```diff
@@ -29,7 +29,7 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.todo',]
+extensions = ['sphinx.ext.todo','rst2pdf.pdfbuilder']
```

Comment:
Is this installed by default with sphinx, or is it an additional requirement? If the latter, probably warrants noting in the README.